ui/src/components/admin/ModelsManager/ModelsManager.js

```diff
@@ -79,21 +47,21 @@ const TableFooterToolbar = styled('div')`
 
 const content = () => {
   return (
-    <Container>
-      <ContentPanel>
-        <MenuBar>
-          <Title>
-            <ModelIcon height={30} width={30} />Model Management
-          </Title>
+    <AdminContainer>
+      <AdminHeader>
+        <Title>
+          <ModelIcon height={30} width={30} />Model Management
+        </Title>
+        <Controls>
           <ControlPill>Add Bulk</ControlPill>
           <ControlPill last>Add A Model</ControlPill>
-        </MenuBar>
-        <Table>
-          <TableHeaderToolbar />
-          <TableFooterToolbar />
-        </Table>
-      </ContentPanel>
-    </Container>
+        </Controls>
+      </AdminHeader>
+      <Table>
+        <TableHeaderToolbar />
+        <TableFooterToolbar />
+      </Table>
+    </AdminContainer>
   );
 };
 
```

Comment:
Consider moving all styled components to the theme directory, either that or I'll move my components into the admin directory, right now we have both approaches which is confusing